frontend/service/Services.ts

```diff
@@ -204,8 +203,6 @@ const getDeployment = async ({
 /**
  * Withdraws the balance of a service
  *
- * @param withdrawAddress Address
- * @param serviceTemplate ServiceTemplate
  * @returns Promise<Service>
```

Comment:
based on return type we don't return Promise<Service> but this? 
`: Promise<{ error: string | null }`